.ci/install.sh

```diff
@@ -60,8 +60,8 @@ else
 fi
 
 # install shfmt
-SHFMT_URL="https://github.com/mvdan/sh/releases/download/v3.0.1/shfmt_v3.0.1_linux_amd64"
-if [ "$(shfmt --version)" != "v3.0.1" ]; then
+SHFMT_URL="https://github.com/mvdan/sh/releases/download/v3.2.0/shfmt_v3.2.0_linux_amd64"
+if [ "$(shfmt --version)" != "v3.2.0" ]; then
```

Comment:
Might as well sneak in a shellcheck update too.

Reply:
I checked, it's still up to date ;).

Reply:
I was going to say I'd swear there's been a new release since February… and there was, but I already updated it in https://github.com/koreader/koreader/pull/6445 for other reasons.